kim/.htaccess

```diff
@@ -6,4 +6,5 @@ RewriteRule ^schulfaecher/(.*) https://skohub.io/dini-ag-kim/schulfaecher/heads/
 RewriteRule ^kmk-vocabs/(.*) https://skohub.io/dini-ag-kim/kmk-vocabs/heads/main/w3id.org/kim/kmk-vocabs/$1 [R=302,L]
 RewriteRule ^license/(.*) https://skohub.io/dini-ag-kim/license/heads/main/w3id.org/kim/license/$1 [R=302,L]
 RewriteRule ^lrmi-profile/(.*) https://dini-ag-kim.github.io/amb/$1 [R=302,L]
+RewriteRule ^educationalLevel/(.*) <https://skohub.io/dini-ag-kim/educationalLevel/heads/main/w3id.org/kim/educationalLevel/$1> [R=302,L]
```

Comment:
This is not a valid URL for redirection. You should remove  "<>"

Reply:
Thank you. I removed the angle brackets